src/EPR.Accreditation.Facade/Profiles/AccreditationProfile.cs

```diff
@@ -13,10 +13,27 @@ public AccreditationProfile()
                 .ForMember(d => d.Contaminents, o => o.MapFrom(s => s.TonnesContaminents))
                 .ForMember(d => d.ProcessLoss, o => o.MapFrom(s => s.TonnesProcessLoss));
 
+            CreateMap<MaterialWasteOutputsDto, MaterialReprocessorDetails>()
```

Comment:
I suspect most of this is already covered too

Reply:
Please see my response to the first comment above.